service/worker/workerdeployment/util.go

```diff
@@ -139,3 +144,18 @@ func DecodeWorkerDeploymentMemo(memo *commonpb.Memo) *deploymentspb.WorkerDeploy
 	}
 	return &workerDeploymentWorkflowMemo
 }
+
+func getSafeDurationConfig(ctx workflow.Context, id string, unsafeGetter func() any, defaultValue time.Duration) (time.Duration, error) {
```

Comment:
this can easily be generic if you want to do that now

Reply:
have to do it later